src/moonstone-samples/src/TooltipSample.js

```diff
@@ -23,10 +24,27 @@ module.exports = kind({
 				]}
 			]
 		},
+		{name: 'sidewaysTooltipButton', kind: TooltipDecorator, style: 'left: 40%; z-index: 1;', components: [
+			{kind: Button, content: 'Sideways Tooltip'},
+			{name: 'sidewaysTooltip', kind: Tooltip, position: 'left top', components: [
+				{name: 'sidewaysTooltipText', content: 'I\'m a tooltip on the side.', style: 'white-space: normal'}
```

Comment:
let's get rid of `content` since it's bound to input value. L22 as well for drag tooltip